homework/04LessonFour.js

```diff
@@ -86,7 +102,7 @@ var nestedObjects = {
 //
 // What is an associative array? What is the difference between an array and an associative array?
 //
-// Your Answer Goes Here:
+// Your Answer Goes Here: Associative arrays (objects) use an explicit index called a key with an associated value. The key can be whatever you want. An array uses an implicit index starting at 0.
```

Comment:
You're on a roll!

![success](https://thumbs.dreamstime.com/x/old-man-celebrating-laptop-16618166.jpg)